fe/fe-core/src/main/java/com/starrocks/server/LocalMetastore.java

```diff
@@ -3276,35 +3294,16 @@ public void createMaterializedView(CreateMaterializedViewStatement stmt)
         LOG.info("Successfully create materialized view[{};{}]", mvName, mvId);
 
         // NOTE: The materialized view has been added to the database, and the following procedure cannot throw exception.
-        if (createMvSuccess) {
-            createTaskForMaterializedView(dbName, materializedView, optHints);
-        }
+        createTaskForMaterializedView(dbName, materializedView, optHints);
```

Comment:
why remove the check of createMvSuccess?

Reply:
because createMvSuccess is always true

Reply:
no, when there is if not exist, the createMvSuccess may be not true.